builder/openstack/step_run_source_server.go

```diff
@@ -39,6 +42,16 @@ func (s *StepRunSourceServer) Run(state multistep.StateBag) multistep.StepAction
 		networks[i].UUID = networkUuid
 	}
 
+	userData := []byte(s.UserData)
```

Comment:
Should we warn if both `UserData` and `UserDataFile` are both given? Or maybe that's just a docs thing.